js/src/components/lineage/PreviewChangeView.tsx

```diff
@@ -296,7 +305,10 @@ export function PreviewChangeView({
               icon={<VscFeedback />}
               variant={"ghost"}
               size={"md"}
-              onClick={() => feedbackToast(true)}
+              onClick={() => {
+                feedbackToast(true);
+                prepareEnvToast(true);
```

Comment:
Show both feedback and prepare env toast when the user clicksthe feedback button seems weird.
`Give us feedback` button should only show a feedback toast.

Reply:
ok, I will remove it.